src/applications/vaos/new-appointment/newAppointmentFlow.js

```diff
@@ -190,9 +197,14 @@ export default {
       return nextState;
     },
   },
+  vaFacilityV2: {
+    url: '/new-appointment/va-facility-2',
+  },
```

Comment:
Add new key for new facility v2 page.  We will add `next` and `previous` keys in a future PR.